cpp/include/raft/handle.hpp

```diff
@@ -148,11 +149,17 @@ class handle_t {
     }
   }
 
-  ///@todo: enable this once we have cuml-comms migrated
-  // void setCommunicator(
-  //   std::shared_ptr<MLCommon::cumlCommunicator> communicator);
-  // const MLCommon::cumlCommunicator& getCommunicator() const;
-  // bool commsInitialized() const;
+  void set_comms(std::shared_ptr<comms::comms_t> communicator) {
```

Comment:
Can a handle hold multiple (sub-)communicators?

Reply:
At the present time, I don’t relieve so, and I think we should discuss what that would like like in a follow-on PR, what do you think?

Reply:
Sure, no need to address everything in a single PR, just want to say that cuGraph needs sub-communicators.

Reply:
I'll open RAFT issues for the feature requests in this PR and someone from cugraph/cuml can knock them out as needed.